source/hdev/Camera/Basic/CaptureHDRCompleteSettings.hdev

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<hdevelop file_version="1.2" halcon_version="19.11.0.0">
+<hdevelop file_version="1.2" halcon_version="19.05.0.0">
```

Comment:
Why this downgrade?

Reply:
Removed it. Halcon auto saves the version I use and I have the older version installed.

Reply:
Doesn't look like you removed it.